grabbit/src/main/groovy/com/twcable/grabbit/GrabbitConfiguration.groovy

```diff
@@ -77,6 +78,20 @@ class GrabbitConfiguration {
                 pathConfigurations
         )
     }
+	
+	private static Collection<String> nonEmptyCollection(Collection<String> excludePaths,ConfigurationException.Builder errorBuilder) {
+		if(excludePaths == null || excludePaths.isEmpty())
+			return excludePaths
```

Comment:
@masroormohammed this return can either return null or an empty collection.  I think you might want to be specific about what you want to return here in the case that there are no exclude paths.